### PR TITLE
Create v-select wrapper that autoselects the only option

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/views/InteractiveMap/CardDialog.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
               ></v-text-field>
             </v-flex>
             <v-flex xs12 md3>
-              <v-select
+              <AutoSelect
```
Comment (lukaw3d) [inline src/views/InteractiveMap/CardDialog.vue]: Open to renaming suggestions. `v-select-auto-single`?

Comment (Midnighter) [reply src/views/InteractiveMap/CardDialog.vue]: `AutoVSelect`?

File: src/components/AutoSelect.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      this.autoselectOnlyOption(vselect);
+    });
+  },
+  methods: {
+    autoselectOnlyOption(vselect: VSelect) {
+      if (vselect.multiple) {
+        throw new Error("AutoSelect does not support multiple selections.");
+      }
+      const firstOption = vselect.allItems[0];
+      const isSelectionEmpty = vselect.selectedItems.length === 0;
```
Comment (Midnighter) [inline src/components/AutoSelect.vue]: I simply don't know: This is always guaranteed to be an array even though you explicitly don't want this for multi selects?

Comment (lukaw3d) [reply src/components/AutoSelect.vue]: Exactly, v-select's internals are generalized like this to support both modes

**Comment by lukaw3d** [comment]
I'll go with v-select-auto

**Pull request merged**